### PR TITLE
issue #99 provide base invoker playbook

### PR DESCRIPTION
**Pull request by domdom82**
issue #99 delete obsolete component playbooks

Signed-off-by: Hoang Anh Le <hoang@de.ibm.com>

### CODE REVIEW

**Comment by jeremiaswerner** [comment]
@domdom82 are the deletions of the loadbalancer, controller, kafka ymls by purpose?

**Comment by domdom82** [comment]
@jeremiaswerner  yes. the component playbooks are now living under /ansible/roles. they were just moved.

File: ansible/roles/invoker/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      "CONSUL_HOST_PORT4": "{{consul.port.http}}"
+      "SELF_DOCKER_ENDPOINT": "{{inventory_hostname}}:{{docker.port}}"
+      "SERVICE_NAME": "invoker{{play_hosts.index(inventory_hostname)}}"
+      "SERVICE_CHECK_HTTP": "/ping"
+      "SERVICE_CHECK_TIMEOUT": "2s"
+      "SERVICE_CHECK_INTERVAL": "15s"
+      "PORT": 8080 
+    volumes:
+      - "{{whisk_logs_dir}}/invoker{{play_hosts.index(inventory_hostname)}}:/logs"
+      - "/var/lib/docker/containers/:/containers"
```
Comment (jeremiaswerner) [inline ansible/roles/invoker/tasks/main.yml]: can you pls make `/var/lib/docker/containers/` configureable. we need that when enabling docker user namespaces

**Comment by domdom82** [comment]
@jeremiaswerner we can certainly do that. but I would handle it in another issue that is related to namespaces. for now we should go with this as it reflects the state of the current build.xml.

File: ansible/roles/invoker/tasks/main.yml (hunk 1 above)
Comment (markusthoemmes) [inline ansible/roles/invoker/tasks/main.yml]: +1 on making this configurable! `/var/lib/docker` is configurable by the docker daemon anyway and might not be in place for every installation.

Comment (domdom82) [inline ansible/roles/invoker/tasks/main.yml]: @jeremiaswerner we can certainly do that. but I would handle it in another issue that is related to namespaces. for now we should go with this as it reflects the state of the current deployInvoker script

Comment (jeremiaswerner) [inline ansible/roles/invoker/tasks/main.yml]: @domdom82 make sense, will open a separate issue.

**Pull request merged**